docs/install.md

```diff
@@ -1,5 +1,7 @@
 # Installation
 
+If PHP is not installed on your environment, install it now; PHP is a requirement for Dock CLI.
```

Comment:
Sure, any suggestions for what the minimum should be? FWIW, PHP 5.x seemed to work fine for me, both in terms of the Docker install and the setup of an Inviqa project using the tool.